ImportExport/ObjectsToPython/ObjectsToPython.FCMacro

```diff
@@ -0,0 +1,344 @@
+# -*- coding: utf-8 -*-
+
+__Name__ = 'Objects To Python'
+__Comment__ = 'Exports objects from a FreeCAD project to a python script'
+__Author__ = 'Christi'
+__Version__ = '0.2'
+__Date__ = '2020-06-06'
+__License__ = 'LGPL-3.0-or-later'
+__Web__ = ''
+__Wiki__ = 'README.md'
+__Icon__ = ''
+__Help__ = ''
+__Status__ = ''
+__Requires__ = 'FreeCAD >= 0.17'
+__Communication__ = 'https://forum.freecadweb.org/viewtopic.php?f=22&t=34612'
+__Files__ = 'ObjectsToPython.py,Readme.md'
+
+VERSION_STRING = __Name__ + ' Macro v' + __Version__
+
+
+import FreeCAD
```

Comment:
Use `import FreeCAD as app` for consistency with other macros and replace `FreeCAD` with `app` in the macro.